Target input required parameter missing in the documentation

Hello,

In the documentation, the example concerning the scan of a Docker image:

```
  - task: trivy@2
    inputs:
      type: 'image'
      image: my.private.registry/org/my-image:latest
```

but there is an error:

```
Starting Trivy task...
##[error]Input required: target
Finishing: Trivy scan on Docker image
```

--- a/docs/trivyv2.md
+++ b/docs/trivyv2.md
@@ -111,7 +111,7 @@ steps:
   - task: trivy@2
     inputs:
       type: 'image'
-      image: my.private.registry/org/my-image:latest
+      target: my.private.registry/org/my-image:latest
 ```
 
 ### Scanning with Aqua Platform support

